Add UI instructions (#455)
kyma-project/istio: docs/user/02-operation-guides/operations/02-20-enable-sidecar-injection.md

Enabling automatic sidecar injection allows `istiod` to watch all Pod creation operations on all Namespaces, which should be part of Istio Service Mesh, and inject the newly created Pods with a sidecar proxy.

> You can enable sidecar proxy injection for either an entire Namespace or a single Deployment. If the sidecar proxy injection is disabled at the Namespace level or the `sidecar.istio.io/inject` label on a Pod is set to `false`, the sidecar proxy is not injected.

Reviewer: ```suggestion
You can enable sidecar proxy injection for either an entire Namespace or a single Deployment. If the sidecar proxy injection is disabled at the Namespace level, or the `sidecar.istio.io/inject` label on a Pod is set to `false`, the sidecar proxy is not injected.
```

> >**WARNING:** Adding the `istio-injection=enabled` label on the Namespace level results in injecting sidecars to all Pods inside of the Namespace. 

Reviewer: why is this cause for a warning, not just a note (or plain text)? Is this problematic? If yes, briefly explain why.

Author: It should be a note

Reviewer: could it even be plain text? Isn't it just a normal sentence that follows the previous one?

> * Follow the steps to enable sidecar proxy injection for a Namespace:

Reviewer: ```suggestion
* To enable sidecar proxy injection for a Namespace, you can use either kubectl or Kyma dashboard:
```

"Follow the steps" is such an empty phrase to me (it's an instructional doc, what else do we expect? :D) ; if possible, I'd like to replace it with more meaningful content

  
  <!-- tabs:start -->

  #### **kubectl**
  
  Use the following command:
  
  ```bash
  kubectl label namespace {YOUR_NAMESPACE} istio-injection=enabled
  ```

  #### **Kyma Dashboard**
  
  1. Select the Namespace where you want to enable sidecar proxy injection.

>   2. Click the **Edit** button.

Reviewer: ```suggestion
  2. Click **Edit**.
```

>   3. In the `UI Form` section, toggle the switch to set the **istio-injection** label value to `enabled` for the Namespace.

Reviewer: ```suggestion
  3. In the `UI Form` section, set the **istio-injection** label value to `enabled` for the Namespace.
```
Again, I'd rather be slightly more abstract than describing the UI details; because they might change

  ![Switch the toggle to enable Istio sidecar injection](./../../../assets/sidecar-injection-toggle-namespace.svg)
  4. Click **Update**.
  <!-- tabs:end -->

> * Follow the steps to enable sidecar proxy injection for a Pod:

Reviewer: ```suggestion
* To enable sidecar proxy injection for a Pod, you can use either kubectl or Kyma dashboard:
```

  <!-- tabs:start -->

  #### **kubectl**
  
  Use the following command:
  
  ```bash
  kubectl patch -n {YOUR_NAMESPACE} deployments/{YOUR_DEPLOYMENT} -p '{"spec":{"template":{"metadata":{"labels":{"sidecar.istio.io/inject":"true"}}}}}'
  ```

  #### **Kyma Dashboard**

  1. Select the Namespace of the Pod's Deployment.

>   2. On the left-hand side sidebar, find the **Workloads** section and select **Deployments**.

Reviewer: ```suggestion
  2. In the **Workloads** section, select **Deployments**.
```

>   3. Select the Pod's Deployment and click the **Edit** button.

Reviewer: ```suggestion
  3. Select the Pod's Deployment and click **Edit**.
```

>   4. In the `UI Form` section, toggle the switch to set the **sidecar.istio.io/inject** label to `true`.

Reviewer: ```suggestion
  4. In the `UI Form` section, set the **sidecar.istio.io/inject** label to `true`.
```
